client/js/components/procedure/StatementSegmentsList/SegmentLocationMap.vue

```diff
@@ -233,6 +237,23 @@ export default {
         })
     },
 
+    /*
+     * Center the map around all drawings and zoom to the combined extent.
+     */
+    setCenterAndExtent () {
+      const extentPolygon = this.$refs.drawPolygon.getExtent()
+      const extentPoint = this.$refs.drawPoint.getExtent()
+      const extentLine = this.$refs.drawLine.getExtent()
+
```

Comment:
can't `this.featuresObject` be used to accomplish thant? As far as I remember, in that var all elements are already combined.

Reply:
`this.featuresObject` only contains the coordinates, but not the extent.

Reply:
:facepalm: I love fitting wording